src/Store.js

```diff
@@ -499,11 +498,12 @@ class Store {
    */
   @action
   deleteLoadingState = (state) => {
-    const querySet = this.loadingStates.get(state.queryTag)
+    const { loadingStates } = this
+    loadingStates.get(state.queryTag)
```

Comment:
looks like this redundant line got left in

Reply:
removed